megamek/src/megamek/common/Engine.java

```diff
@@ -450,7 +450,11 @@ public int integralHeatSinkCapacity(boolean compact) {
      */
     public String getShortEngineName() {
         if (engineType < TYPE_KEYS.length) {
-            return String.format("%d%s", engineRating, Messages.getString("Engine." + TYPE_KEYS[engineType]));
+            if (hasFlag(SUPPORT_VEE_ENGINE)) {
+                return String.format("%s", Messages.getString("Engine." + TYPE_KEYS[engineType])).trim();
```

Comment:
No need to call String.format.
```suggestion
                return Messages.getString("Engine." + TYPE_KEYS[engineType]).trim();
```